numba_mpi/mpi.py

```diff
@@ -196,7 +196,7 @@ def recv(data, source, tag):
 
     buffer = (
         data if data.flags.c_contiguous
-        else np.empty_like(data)
+        else np.empty(data.shape, data.dtype)
```

Comment:
would `np.empty_like(data, order='C')` be more readable?

Reply:
Yes, I think this would be more expressive.